packages/libs/react-components/src/components/Input/InputGroup.tsx

```diff
@@ -0,0 +1,67 @@
+import { AlertBox } from '../Icons/svgs/AlertBox';
+import { AlertCircleOutline } from '../Icons/svgs/AlertCircleOutline';
+
+import {
+  StyledHelper,
+  StyledInfo,
+  StyledInputGroupHeader,
+  StyledInputGroupWrapper,
+  StyledInputs,
+  StyledLabel,
+  StyledTag,
+} from './styles';
+
+import classnames from 'classnames';
+import React, { FC } from 'react';
+
+export interface IInputGroupProps {
+  label?: string;
+  tag?: string;
+  info?: string;
+  helper?: string;
+  status?: 'success' | 'error';
+  disabled?: boolean;
+  children: React.ReactNode;
+  className: string;
+}
+
+export const InputGroup: FC<IInputGroupProps> = ({
+  label,
+  tag,
+  info,
+  helper,
+  children,
+  status,
+  disabled,
+  className,
+  ...rest
```

Comment:
I dont think we need rest props right? Because we have specified which props are allowed